test/travis_ci/install_ubuntu_14_04.sh

```diff
@@ -22,4 +22,4 @@ wget https://cmake.org/files/v3.5/cmake-3.5.2-Linux-x86_64.tar.gz
 sudo tar -C /usr -x -z -f cmake-3.5.2-Linux-x86_64.tar.gz --strip-components=1
 
 # install data files.
-svn co svn://missmarple.lbl.gov/work3/teca/TECA_data@${TECA_DATA_REVISION} TECA_data
+svn co svn://svn.code.sf.net/p/teca/TECA_data@${TECA_DATA_REVISION} TECA_data
```

Comment:
can you be sure that there is a newline at the end of the last line, also note the same issue w/ the ubuntu shell script